android-components/components/feature/fxsuggest/src/main/java/mozilla/components/feature/fxsuggest/FxSuggestStorage.kt

```diff
@@ -31,7 +32,10 @@ class FxSuggestStorage(context: Context) {
     // does I/O, so `store.value` should only be accessed from the read or write scope.
     @VisibleForTesting(otherwise = VisibleForTesting.PRIVATE)
     internal val store: Lazy<SuggestStore> = lazy {
-        SuggestStore(File(context.cacheDir, DATABASE_NAME).absolutePath)
+        SuggestStoreBuilder()
+            .cachePath(File(context.cacheDir, DATABASE_NAME).absolutePath)
+            .dataPath(context.getDatabasePath(DATABASE_NAME).absolutePath)
```

Comment:
I copied this from the logins component and I'm pretty sure this is the correct way to get a database path, but I'm no android engineer.  @linabutler How confident are you on this stuff?  Should we flag someone from the android team or do you feel good reviewing it?

Reply:
LGTM! I love this new builder API 🤩

`context.getDatabasePath(...).absolutePath` is what we were using before we switched to using the temporary cache (#4143), so I feel good about reviewing this!

One thing I am wondering about, though—should we use different names for the two databases, and rename the constants to match?

We can keep the cache one named `"suggest.sqlite"`, so that we don't trigger a re-ingest, but let's maybe change the name of the constant to something like `CACHE_DATABASE_NAME`, and add a new constant with a different name for the data database? I'm just thinking that it might be a little confusing to have two databases named `suggest.sqlite` in different directories and with different contents.